starters/basic-starter/components/consumer-image-style-media--image.tsx

```diff
@@ -0,0 +1,42 @@
+import Image from 'next/image';
+
+import { MediaImageProps } from './media--image';
+
+export function ConsumerImageStyleMediaImage({
```

Comment:
This is not exactly what I was thinking. What I was thinking was allowing to use the pre-existing `<MediaImage>` component for rendering an image using an image style. Maybe it could be an additional argument like `imageStyle` which would then render the image using the image style